_episodes/12-cmdline.md

```diff
@@ -696,7 +696,7 @@ the program now does everything we set out to do.
 > >     assert len(sys.argv) == 4, 'Need exactly 3 arguments'
 > >
 > >     operator = sys.argv[1]
-> >     assert operator in ['add', 'subtract', 'multiply', 'divide'], \
+> >     assert operator in ['--add', '--subtract', '--multiply', '--divide'], \
 > >         'Operator is not one of add, subtract, multiply, or divide: bailing out'
```

Comment:
To prevent confusion, do you think we should have the dashes in the error message as well?
```suggestion
> >         'Operator is not one of --add, --subtract, --multiply, or --divide: bailing out'
```